src/encoded/tests/data/master-inserts/organism.json

```diff
@@ -0,0 +1,43 @@
+[
```

Comment:
Good move to add these to master-inserts.  You can remove the inserts/organism.json then right?

Reply:
Done